frontend/src/App.vue

```diff
@@ -1,21 +1,24 @@
 <template>
   <div>
-    <header>
-      <nav v-if="!$route.path.includes('/login')" class="bg-blue-600 p-4 shadow-md">
-        <ul class="flex space-x-4">
-          <li>
-            <RouterLink to="/" class="text-white font-bold hover:underline">Home</RouterLink>
-          </li>
-          <li>
-            <RouterLink to="/arrangementrequests" class="text-white font-bold hover:underline">Arrangement Requests</RouterLink>
-          </li>
-          <li>
-            <RouterLink to="/submittedview" class="text-white font-bold hover:underline"> Go to submitted requests</RouterLink>
-          </li>
-        </ul>
-      </nav>
+
+    <header v-if="!$route.path.includes('/login')">
+      <NavBar :userRole="userRole" />
     </header>
     <router-view />
   </div>
 </template>
+<script>
+import NavBar from './components/NavBar.vue';
+
+export default {
+  components: {
+    NavBar
+  },
+  data(){
+    return {
+      userRole: 'HR'
```

Comment:
can we fetch the role from local storage instead of hardcoding the user role? 

you can see utils/localStorage.js, theres a method to fetch the data thats required by passing in the key